tests/frontend/global-setup.js

```diff
@@ -0,0 +1,5 @@
+module.exports = async (config) => {
+  process.env.REACT_APP_URL = "http://localhost:3000";
+  process.env.BACKEND_URL = "http://localhost:3001";
```

Comment:
I think we do since we both test the frontend at 3000 and we also listen to request that the frontend sends to the backend at 3001 in order to validate and mock them